### PR TITLE
Bug 1608874 - Enable fennec-production on the mobile instances

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Follows https://github.com/mozilla-releng/scriptworker-scripts/pull/144 up.

### CODE REVIEW

File: pushapkscript/docker.d/init_worker.sh [hunk 1]
```diff
@@ -110,17 +112,20 @@ case $COT_PRODUCT in
...
         export GOOGLE_CREDENTIALS_FIREFOX_BETA_PATH=$CONFIG_DIR/beta.p12
+        export GOOGLE_CREDENTIALS_FIREFOX_RELEASE_PATH=$CONFIG_DIR/release.p12
 
         echo $GOOGLE_CREDENTIALS_FENIX_NIGHTLY | base64 -d >     $GOOGLE_CREDENTIALS_FENIX_NIGHTLY_PATH
         echo $GOOGLE_CREDENTIALS_FENIX_BETA | base64 -d >        $GOOGLE_CREDENTIALS_FENIX_BETA_PATH
         echo $GOOGLE_CREDENTIALS_FENIX_PROD | base64 -d >        $GOOGLE_CREDENTIALS_FENIX_PROD_PATH
         echo $GOOGLE_CREDENTIALS_FOCUS | base64 -d >             $GOOGLE_CREDENTIALS_FOCUS_PATH
         echo $GOOGLE_CREDENTIALS_REFERENCE_BROWSER | base64 -d > $GOOGLE_CREDENTIALS_REFERENCE_BROWSER_PATH
-        echo $GOOGLE_CREDENTIALS_FIREFOX_AURORA | base64 -d >      $GOOGLE_CREDENTIALS_FIREFOX_AURORA_PATH
+        echo $GOOGLE_CREDENTIALS_FIREFOX_AURORA | base64 -d >    $GOOGLE_CREDENTIALS_FIREFOX_AURORA_PATH
```
Comment (MihaiTabara) [inline pushapkscript/docker.d/init_worker.sh:122]: Thanks for fixing this nit 👍

File: pushapkscript/docker.d/worker.yml [hunk 2]
```diff
@@ -72,10 +72,17 @@ products:
               package_names: [ "org.mozilla.firefox_beta" ]
               certificate_alias: 'fennec-beta'
               google:
-                # TODO Change to "production" once the Fenix Beta migration is done
-                default_track: 'alpha'
+                default_track: 'production'
```
Comment (MihaiTabara) [inline pushapkscript/docker.d/worker.yml]: @JohanLorenzo I reckon so far we've been publishing to `alpha` track and RelMan/Mobile folks have promoted to `production`. If this goes to production, we're performing this step automatically. Are they expecting this, should we heads-up them? I'm wondering what the cadence for Fenix beta is these days.

Unrelatedly, kudos for all these comments left in code, I feel much safer reading some of the TODOs knowing I've reviewed some of these things in the past.

File: pushapkscript/docker.d/worker.yml [hunk 3]
```diff
@@ -72,10 +72,17 @@ products:
...
                 service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_BETA" }
                 credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FIREFOX_BETA_PATH" }
+            fennec-production:
+              package_names: [ "org.mozilla.firefox" ]
+              certificate_alias: 'fennec-production'
```
Comment (MihaiTabara) [inline pushapkscript/docker.d/worker.yml:81]: This matches the certificate import in the docker.d file 👍

**Review by MihaiTabara — APPROVED** [review]
LGTM!

File: pushapkscript/docker.d/worker.yml (hunk 2 above)
Comment (JohanLorenzo) [reply pushapkscript/docker.d/worker.yml]: Yeah, I wanted to ask on Friday's call, but I forgot. I'll get the approval first!

Comment (JohanLorenzo) [reply pushapkscript/docker.d/worker.yml]: We want to keep the alpha track for now. Full conversation at https://mozilla.slack.com/archives/CKM7DLL67/p1591024730316900

**Comment by rail** [comment]
Looks like the production push is b0rken :)
`error: GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_RELEASE is not set`

Probably we need to update cloudops-infra/SOPS?

**Comment by MihaiTabara** [comment]
> Looks like the production push is b0rken :)
> `error: GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_RELEASE is not set`
> 
> Probably we need to update cloudops-infra/SOPS?

Fixed SOPS, should be good now, I've repushed master to production pushapk branch.

**Pull request merged**